src/test/java/seedu/travel/ui/ExpandedPlacePanelTest.java

```diff
@@ -0,0 +1,61 @@
+package seedu.travel.ui;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+
+import seedu.travel.model.place.Photo;
+import seedu.travel.model.place.Place;
+import seedu.travel.testutil.Assert;
+import seedu.travel.testutil.PlaceBuilder;
+
+public class ExpandedPlacePanelTest extends GuiUnitTest {
+
+    public static final String INVALID_PATH = "C://";
+    public static final String EMPTY_PATH = "pBSgcMnA";
+
+    @Test
+    public void equals() {
+        Place place = new PlaceBuilder().build();
+        ExpandedPlacePanel placePanel = new ExpandedPlacePanel(place);
+
+        // same place -> returns true
+        ExpandedPlacePanel copy = new ExpandedPlacePanel(place);
+        assertTrue(placePanel.equals(copy));
+
+        // same object -> returns true
+        assertTrue(placePanel.equals(placePanel));
+
+        // null -> returns false
+        assertFalse(placePanel.equals(null));
```

Comment:
If possible, try not to use .equals to compare to null, maybe try placePanel == null instead?

Reply:
Unfortunately, this test targets the overwritten equals() method in ExpandedPlacePanelTest, so I will have to invoke the equals() method.

Reply:
Ah I see, okay thanks for clarifying